saleor/checkout/capture_payment.py

```diff
@@ -0,0 +1,33 @@
+from prices import Money
+
+from saleor.order import OrderStatus
+from saleor.order.models import Order
+from saleor.payment import gateway
+from saleor.plugins.manager import PluginsManager
+
+
+def capture_payment(order, payment, manager, to_pay):
+    if payment and payment.can_capture():
+        amount = min(to_pay, Money(payment.get_charge_amount(), payment.currency))
+        gateway.capture(
+            payment,
+            manager,
+            channel_slug=order.channel.slug,
+            amount=amount,
+        )
+        return amount
+
+
+def capture_payments(order: Order, manager: PluginsManager):
+    order.status = OrderStatus.UNFULFILLED
+    order.save(update_fields=["status"])
```

Comment:
IMO the status should be updated at the end of this method. I suppose we do not want to update order status when capture failed.